crates/types/src/logs/metadata.rs

```diff
@@ -235,18 +248,43 @@ impl Chain {
     }
 }
 
+/// Creates appropriate [[LogletParams]] value that can be used to start a fresh
```

Comment:
Is there a difference between `[[LogletParams]]` and ``[`LogletParams`]``? So far we mostly used the latter in Rust docs, I believe.

Reply:
No difference, will fix to make it consistent.